### ISSUE TITLE
git commit -m

### ISSUE THREAD

**Issue by ValentinGaudin** [withfig/fig#2550]
### Checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have run `fig restart` and replicated the issue again

### Operating system

macOS 13.2.1 (22D68)

### Expected behaviour

When run git commit -m in fig, i excepted like before to have `git commit -m ''`

### Actual behaviour

When i run  command now,  i just have git commit -m without quote

### Environment

```yaml
fig-details:
  - 2.16.0
hardware-info:
  - model: 
  - model-id: 
  - chip-id: Apple M1 Pro
  - cores: 8
  - mem: 16.00 GB
os-info:
  - macOS 13.2.1 (22D68)
environment:
  - shell: /bin/zsh
  - terminal: iterm
  - cwd: /Users/gaudinvalentin/xCoding/xTCM/icfc-plateforme-conseils
  - exe-path: /Users/gaudinvalentin/.fig/bin/fig
  - install-method: unknown
  - env-vars:
    - FIGTERM_SESSION_ID: 24a859a5-b780-454e-aac4-53954196973c
    - FIG_SET_PARENT_CHECK: 1
    - FIG_TERM: 2.16.0
    - PATH: /opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin:/Users/gaudinvalentin/.fig/bin:/Users/gaudinvalentin/.local/bin:/Users/gaudinvalentin/Library/Application Support/JetBrains/Toolbox/scripts:/Users/gaudinvalentin/.fzf/bin
    - SHELL: /bin/zsh
    - TERM: xterm-256color
    - __CFBundleIdentifier: com.googlecode.iterm2
    - FIG_PID: 3711
    - FIG_SET_PARENT: 24a859a5-b780-454e-aac4-53954196973c
```

**Comment by grant0417** [comment]
Hey @ValentinGaudin, the spec got changed recently which dropped the quotes from being added by default. 

@mschrage any thoughts on how this could be resolved if people want to keep using the `-m ''` option?

**Comment by ValentinGaudin** [comment]
Hey @grant0417 thank you for letting me know for the new spec. 

Well i also want to know if we can have a option to keep using the` -m ''` option :)

**Comment by ZhaoKunLong** [comment]
Yep. it's really useful when `git commit -m ' '` have the quote. less two times keypress 😄 . Every day I almost commit 30 times. haha. less 60 times keypress.....

**Comment by programmer4229** [comment]
I am currently working on this issue

**Comment by programmer4229** [comment]
I think I have fixed it but this is my first time contributing to this project and I do not know how to create a PR since the changes that needed to be made are within the autocomplete repo not this one. Any advice?
